browser/resources/settings/brave_default_extensions_page/brave_default_extensions_page.js

```diff
@@ -20,7 +20,19 @@ Polymer({
     showRestartToast_: Boolean,
     torEnabled_: Boolean,
     widevineEnabled_: Boolean,
-    disableTorOption_: Boolean
+    disableTorOption_: Boolean,
+    udEnabled_: Boolean,
```

Comment:
please don't abbreviate, it makes it much harder to find things when searching.

Reply:
Fixed in https://github.com/brave/brave-core/pull/7702/commits/7b47bd19d16b8c898fe3a160b02effaa4cb59e5b